src/fileConvert/fileConvert.ts

```diff
@@ -40,18 +45,25 @@ export const configureFile = async (filename: string) => {
 
     data.splice(0, 0, headerLine);
     data.splice(data.length, 0, trailerLine);
+    logger.debug('Spliced data into the csv file');
```

Comment:
I think we should change these debugs to informations. In prod we would be running with an information log level and we would have limited information to troubleshoot and issues.